DataScience/DashboardMpi/dashboard_e2e.py

```diff
@@ -141,14 +148,18 @@ def dashboard_e2e(args, delta_mod_t=3600, max_connections=50):
         best = sweep.sweep(multi_grid, env, base_command)
 
         predict_opts = {'#base': '--cb_explore_adf --epsilon 0.2 --compressed --dsjson --save_resume --preserve_performance_counters'}
```

Comment:
again, let's put this has part of the else clause below. Also, let's build the #base string incrementally: i.e. the only difference between cb and ccb, is c(c)b_explore_adf, so that's the only thing you need in the if else clause. Then you add "--epsilon 0.2... etc etc"

Reply:
yeah, you can split it into thing like: predict_opts = {'#algo': '--cb_explore_adf --epsilon 0.2', '#other': '--compressed --dsjson --save_resume --preserve_performance_counters'}

Reply:
no, I don't want to change the structure of the dict. What I had in mind is:

```
if log_type == 'ccb':
        base_cmd_str = '--ccb_explore_adf --epsilon 0'
else:
        base_cmd_str = '--cb_adf'
base_command = {'#base': base_cmd_str+' --dsjson --compressed --save_resume --preserve_performance_counters'
```
I think this is the cleaner way to maintain. You immediately see what are the CB vs CCB changes. Do you guys agreee?